test/regexp.test.js

```diff
@@ -27,28 +28,31 @@ describe('cloudant regexp', function() {
   });
   it('find all foos begining with b', function(done) {
     Foo.find({where: {bar: {regexp: '^b'}}}, function(err, entries) {
-      console.log (entries);
+      if (err) return done(err);
+      console.log(entries);
       entries.should.have.lengthOf(1);
       entries[0].bar.should.equal('b');
       done();
     });
   });
   it('find all foos like b', function(done) {
     Foo.find({where: {bar: {like: 'b'}}}, function(err, entries) {
-      console.log (entries);
+      if (err) return done(err);
+      console.log(entries);
```

Comment:
Eslint doesn't only complain syntax error but also errors like "error are supposed to be handled", this is a fix for error not handled. I am not moving this down 2 lines, I am adding a new line `if (err) return done(err)`